### PR TITLE
Added a few more common toppings.

### PR DESCRIPTION
**Pull request by webmaven**

### CODE REVIEW

File: data/foods/pizzaToppings.json [hunk 1]
```diff
@@ -4,11 +4,16 @@
     "anchovies",
     "artichoke",
     "bacon",
+    "breakfast bacon",
+    "canadian bacon",
```
Comment (hugovk) [inline data/foods/pizzaToppings.json]: @webmaven How about "Canadian bacon"? For people who want everything in lowercase they can change it themselves, but it's difficult to add caps when there are none in the source data to begin with.

Comment (dariusk) [inline data/foods/pizzaToppings.json]: That's a good idea. @webmaven, can you capitalize "Canadian" and then I'll
merge?

**Comment by webmaven** [comment]
Done?

**Comment by dariusk** [comment]
Thank you!

**Pull request merged**